docs/qa/environments.rst

```diff
@@ -66,11 +60,10 @@ for testing admin frontends to Normandy, and always has the latest  code. Someti
 is broken. Dev doesn't have a separate read-only and admin server, there is only one
 server.
 
-- **Legacy admin:** https://normandy.dev.mozaws.net/
-- **Bearer auth admin:** https://normandy-admin-bearer.dev.mozaws.net/
+- **Read-only API server:** https://dev.normandy.nonprod.cloudops.mozgcp.net/
+- **Writable API server:** https://dev-admin.normandy.nonprod.cloudops.mozgcp.net/
```

Comment:
Why are these called "Read-only" and "Writable" but for Prod and Stage the nomenclature "Public" and "Admin" are used?

Perhaps I'm too invested in Normandy to speak but I like the words "Read-only/Writable" much better than "Public/Admin".

Reply:
The main difference is that Stage and Prod have VPN restrictions on their admin server, whereas Dev just has two servers, one read-only and one write-only. They are both "public" though.

Dev is different than stage and prod, which is why it is labeled differently. I realize now that the prose above these URLs also needs updated. I'll re-do this section.

Reply:
Feel free to ignore me actually. I was just trying to dump my reaction to what I was reading. It's not wrong. Just a point that, from my perspective, it can be improved slightly. 

In a sense, both are "admin APIs". You can use the "Read-only API" to get insights that aren't critical to the Firefox Normandy clients. 

All I'm saying is that "Writeable/Read-only" are *better* words than "Public/Admin". 

Bla. This almost beyond bikeshedding.